doc/man3/ASYNC_start_job.pod

```diff
@@ -146,6 +154,14 @@ occur.
 Some platforms cannot support async operations. The ASYNC_is_capable() function
 can be used to detect whether the current platform is async capable or not.
 
+Custom memory allocation functions are supported for the POSIX platform.
+Custom memory allocation functions allow alternative methods of allocating
+stack memory such as mmap, or using stack memory from the current thread.
+Using an ASYNC_stack_alloc_fn callback also allows manipulation of the default
+stack size. The stack size can be altered by allocating a stack of a size
```

Comment:
Should we mention what the default stack size is here?

Reply:
Updated to mention that the default is 32k